### ISSUE TITLE
Deploy command is slightly different

### ISSUE THREAD

**Issue by mattmenefee** [jlengstorf/roots.code.lengstorf.com#2]
Awesome guide dude! Thanks! One small change that I noticed, the deploy command you have specified didn't work:

```shell
./deploy.sh production [...]
```
 
Instead, this worked ([from the docs](https://roots.io/trellis/docs/deploys/)):

```shell
./bin/deploy.sh production [...] # notice the addition of "bin"
```

**Comment by 7hr1LL** [comment]
I can only agree- awesome guide!! saved me hours! 
Thats true. I think the guide refers to the last Trellis version.

**Comment by jlengstorf** [comment]
Yeah, it looks like the Trellis core updated since I made this tutorial. This site uses v1.0.0-rc.1, which is the latest release, but it appears they've made some commits since then (no releases, though).

I just pushed an [update to the article](https://code.lengstorf.com/learn-trellis-wordpress-roots/#part-vii-deploy-the-site-to-production) with the updated command and a note.

Thanks for the heads up!
